### DISCUSSION TITLE
Awesome library - Adding an example to PyVista

### DISCUSSION THREAD

**Discussion by akaszynski** [magpylib/magpylib#575] [General]
Great work here, really excited to use this for visualization demos for PyVista.

https://user-images.githubusercontent.com/11981631/213574611-9664ad46-a6f1-4f7b-9277-1697c09cd282.mp4

**Comment by OrtnerMichael** [comment]
@akaszynski 
This is very nice !!! Are you willing to share the code that creates this demo ?

**Reply by akaszynski** [reply]
Yes, sorry for the delay. It's still a PR, should be out for 0.39.

Here's the PR: https://github.com/pyvista/pyvista/pull/3058
